tests/models/models_test.py

```diff
@@ -109,7 +109,9 @@ def test_save_and_load_convtasnet(fb, sample_rate):
 
 
 def test_dprnntasnet_sep():
-    nnet = DPRNNTasNet(n_src=2, n_repeats=2, bn_chan=16, hid_size=4, chunk_size=20, n_filters=32)
+    nnet = DPRNNTasNet(
+        n_src=2, n_repeats=2, bn_chan=16, hid_size=4, chunk_size=20, n_filters=32, use_mulcat=True
```

Comment:
It's not what I meant. 
I mean to use the `@pytest.mark.parameterize` to test both. 
I'll take care of it.